### PR TITLE
Fix collection mobile

### PR DESCRIPTION
**Pull request by mikeyrayvon**
fix image layout for single collection on mobile.

this is already deployed to production.......

### CODE REVIEW

File: css/responsive/mobile.styl [hunk 1]
```diff
@@ -48,7 +48,6 @@ input, select, button, .button, textarea, input[type="checkbox"], input[type=rad
 body.single-collection
   width: 100%
   height: 100%
-  overflow-x: hidden
```
Comment (varffs) [inline css/responsive/mobile.styl:51]: why is this removed?

File: css/responsive/mobile.styl [hunk 2]
```diff
@@ -75,12 +74,14 @@ body.single-collection
...
 .single-collection-row
+  display: block
+  text-align: center
   justify-content: center
 
 .single-collection-item
   flex: 0 1 auto
-  max-height: 100%
-  width: auto
+  max-height: none
```
Comment (varffs) [inline css/responsive/mobile.styl:83]: is this just to allow a scroll if needed?

File: js/main.js [hunk 3]
```diff
@@ -196,14 +196,15 @@ Site.Product = {
 };
 
 Site.Layout = {
-  windowWidth: $(window).width(),
-  windowHeight: $(window).height(),
   init: function() {
     var _this = this;
 
+    _this.windowWidth = $(window).width();
+    _this.windowHeight = $(window).height();
```
Comment (varffs) [inline js/main.js:203]: I can't see that these values are updated on resize, but might not be clear in the github diff view

File: js/main.js [hunk 4]
```diff
@@ -212,16 +213,20 @@ Site.Layout = {
   },
 
   convertScroll: function() {
-    if ($('body').css('overflow-x') == 'scroll') {
+    var _this = this;
+
+    if ($('body').css('overflow-x') == 'scroll' && _this.windowWidth >= 720) {
```
Comment (varffs) [inline js/main.js:218]: should the `720` value here come from a variable on `Site` as has happened on some sites that need a mobile threshold?

File: css/responsive/mobile.styl (hunk 2 above)
Comment (mikeyrayvon) [reply css/responsive/mobile.styl:83]: if there is max height the items will squish on landscape

File: js/main.js (hunk 4 above)
Comment (mikeyrayvon) [reply js/main.js:218]: sure go for it

File: css/responsive/mobile.styl (hunk 1 above)
Comment (mikeyrayvon) [reply css/responsive/mobile.styl:51]: wouldn't scroll otherwise

Comment (mikeyrayvon) [reply css/responsive/mobile.styl:51]: chrome issue i guess

**Comment by mixcas** [comment]
Why am i asigned to this? @varffs fix/change whtever is needed and merge

**Comment by mikeyrayvon** [comment]
because he doesnt have a local @mixcas
